src/components/organisms/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,98 @@
+import DesktopNavigation from "@components/organisms/Navbar/DesktopNavigation";
+import { useEffect, useState } from "react";
+import NavbarLogo from "./components/logo";
+import Profile from "./components/profile";
+import MobileDropdown from "./MobileDropdown";
+import { styled } from "@stitches/react";
+import { createStyles } from "@theme";
+
+const Navbar = () => {
+  const [color, setColor] = useState("transparent");
+  const [shadow, setShadow] = useState("0 0.1px 0.2px #0d0d0d");
+
+  const changeNavbarColor = () => setColor("#fff");
```

Comment:
lots of the navbar logic can probably be put in a custom hook.

https://reactjs.org/docs/hooks-custom.html